webapp/components/admin_console/recycle_db.jsx

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+
+import Client from 'utils/web_client.jsx';
+import * as Utils from 'utils/utils.jsx';
+
+import {FormattedMessage} from 'react-intl';
+
+export default class RecycleDbButton extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.handleRecycle = this.handleRecycle.bind(this);
+
+        this.state = {
+            loading: false,
+            fail: null
+        };
+    }
+
+    handleRecycle(e) {
+        e.preventDefault();
+
+        this.setState({
+            loading: true,
+            fail: null
+        });
+
+        Client.recycleDatabaseConnection(
```

Comment:
Preferably wait until https://github.com/mattermost/platform/pull/3110 is merged, then follow that as an example